js/Particles.js

```diff
@@ -8,7 +8,8 @@ PHOTONS.RangeType = Object.freeze( {
 
 	Cube: 1,
 	Sphere: 2,
-	Plane: 3
+	Plane: 3,
+	Integer: 4

```

Comment:
Hmmm, I'm thinking we should go with either "Default" or something like "Singular". I'm honestly not totally sold on either of those though, so if you think of something better, let me know :)